src/test/kotlin/no/nav/syfo/unntak/api/UnntakApiSpek.kt

```diff
@@ -22,7 +23,7 @@ class UnntakApiSpek : Spek({
     val objectMapper: ObjectMapper = configuredJacksonMapper()
     val urlUnntakPersonIdent = "$unntakApiBasePath/$unntakApiPersonidentPath"
 
-    describe("${UnntakApiSpek::class.java.simpleName}: Create unntak for person") {
+    describe(UnntakApiSpek::class.java.simpleName) {
```

Comment:
Hvorfor ikke ha med en beskrivelse av hva klassen skal gjøre?

Reply:
Ja, det må i så fall bli en beskrivelse av hva hele apiet gjør, men det blir kanskje bare for generisk til at det gir noe verdi 🤷

Reply:
Ja, blir i så fall å gjenta det under, så tenker det ikke gir så mye verdi i dette tilfellet.